### PR TITLE
Adding GSoC'22 final report

### PR DESCRIPTION
**Pull request by xtanion**
[WIP] I was not able to generate HTML due to the following issue `Could not import extension matplotlib.sphinxext.plot_directive`.

### CODE REVIEW

**Comment by skoudoro** [comment]
you need to update you matplotlib version I suppose

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+   .. figure:: https://user-images.githubusercontent.com/74976752/174492510-b9f10816-3058-4a7b-a260-0627406354ba.png
+      :alt: image
+
+
+   *Pull Requests:*
+
+   -  **Importing glTF files: (Merged)**
+      https://github.com/fury-gl/fury/pull/600
+   -  **Loading glTF Demo (with textures) (Merged):**
+      https://github.com/fury-gl/fury/pull/600
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:87]: You should avoid tabulation in the whole document. See below how it looks like due to the tabulations: 

image

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+
+
+Exporting Scene as a glTF
+*************************
+
+
+   The fury scene can contain multiple objects such as actors, cameras,
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:94]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+   -  **Exporting scene as glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+   -  **Exporting scene as glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+
+
+Simple Actor Animations
+***********************
+
+   Added simple actor animations (translation, rotation & scale of
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:111]: same as above for the tabulations. Maybe this is why videos integrations does not work:

image

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+   actors) support. The animation data (transformation and timestamp) is
+   stored in buffers. It converts the binary data to ndarrays and
+   creates a timleline for each animation in glTF animations. This
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:114]: typo: `timeline`

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+   timeline containsactors an can be added to the scene. We can animate
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:115]: typo: missing space `containsactors`

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+   the scene by updating timline inside a timer callback.
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:116]: typo: timline

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+   -  **Simple Animations in glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+   -  **Simple Animations in glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+
+
+Morphing in glTF
+****************
+
+   glTF allows us to animate meshes using morph targets. A morph target
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:131]: you should not start with tabulations. See above. Maybe this is why videos integrations does not work: 

image

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+   -  **Morphing support in glTF: (Under Review)**
+      https://github.com/fury-gl/fury/pull/700
+   -  **Morphing in glTF demo: (Under Review)**
+      https://github.com/fury-gl/fury/pull/700
+
+
+Skeletal Animations (Skining)
+*****************************
+
+   Another way of animating a glTF is by skinning. It allows the
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:151]: you should not start with tabulations. See above

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+   skinning implementation works with most of the glTF sample modes, It
+   struggles with models that have multiple actors (e.g. BrainStem). It
+   can be fixed by using the vertex shader to update the vertices. The
+   current implementation of skinning supports the following:
+
+   -  Multiple animation support
+   -  Multiple node and multiple actor animation with textures
+   -  Show or hide bones/skeleton of the model.
+
+   https://user-images.githubusercontent.com/74976752/194862048-7ce65b42-2717-436b-b311-85368f3c3714.mp4
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:168]: vidéos integrations does not work:

image

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 10]
```diff
@@ -0,0 +1,406 @@
...
+      https://github.com/fury-gl/fury/pull/685
+
+Objectives in Progress
+----------------------
+
+
+PBR and emission materials in glTF
+**********************************
+
+   The glTF format supports Physically based rendering also. PBR allow
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:184]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 11]
```diff
@@ -0,0 +1,406 @@
...
+   different representations of these physical material properties. One
+   that is frequently used is the metallic-roughness-model. We have
+   various material properties already in FURY, we need to apply it to
+   glTF models as well.
+
+
+Skinning for models with no indices
+***********************************
+
+   The glTF format supports non-indexed geometry (e.g., the ``Fox``
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:197]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 12]
```diff
@@ -0,0 +1,406 @@
...
+
+   -  Downloading multiple models asynchronously.
+   -  Get the path to the downloaded model using it   -  Download any model using the URL of the model.
+
+   *Pull Requests:*
+
+   -  **Fetching glTF sample models from github: (Merged)**
+      https://github.com/fury-gl/fury/pull/602
+   -  **Fixing github API limit: (Merged)**
+      https://github.com/fury-gl/fury/pull/616
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:230]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 13]
```diff
@@ -0,0 +1,406 @@
...
+      `fury-gl/fury/#533 <https://github.com/fury-gl/fury/pull/533>`__
+   -  **Cone actor uses repeat primitive by default**:
+      `fury-gl/fury/#547 <https://github.com/fury-gl/fury/pull/547>`__
+   -  **Updated code of viz_network_animated to use fury.utils**:
+      `fury-gl/fury/#556 <https://github.com/fury-gl/fury/pull/556>`__
+   -  **Added simulation for Tesseract**:
+      `fury-gl/fury/#559 <https://github.com/fury-gl/fury/pull/559>`__
+   -  **GLTF actor colors from material**
+      `fury-gl/fury/#689 <https://github.com/fury-gl/fury/pull/689>`__
+
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:246]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 14]
```diff
@@ -0,0 +1,406 @@
...
+
+GSoC weekly blogs
+*****************
+
+   -  My blog posts can be found on the `FURY
+      website <https://fury.gl/latest/blog/author/Shivam-Anand.html>`__
+      and the `Python GSoC
+      blog <https://blogs.python-gsoc.org/en/xtanions-blog/>`__.
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:254]: same as above for the tabulations

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst [hunk 15]
```diff
@@ -0,0 +1,406 @@
...
+
+Timeline
+--------
+
+
++---------------------------+-----------------+------------------------+
+| Date                      | Description     | Blog Link              |
++===========================+=================+========================+
+| Week 0(24-05-2022)        | My Journey to   | `F                     |
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-02-01-final-report-shivam.rst:263]: blog link does not work, to update:

image

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

Thank you for this first version. Please, can you address all issue below. We can not merge this PR as it is. this is mainly reformatting text and links + some typo.

Otherwise, the content looks good. Thank you

File: docs/source/posts/2022/2023-02-01-final-report-shivam.rst (hunk 1 above)
Comment (xtanion) [reply docs/source/posts/2022/2023-02-01-final-report-shivam.rst]: Done @skoudoro , PTAL
Thanks

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

Thank you for the update, it is much better. 

is it possible to generate GIF instead of videos for your blog posts?  because, it is really hard to see your videos, see below:

image

Thanks !

**Comment by xtanion** [comment]
Done @skoudoro

**Comment by skoudoro** [comment]
Perfect! thank you @xtanion, merging

**Pull request merged**